### DISCUSSION TITLE
How to play a sound when a new chat message appears

### DISCUSSION THREAD

**Discussion by Bizarrebra** [Chatterino/chatterino2#2662] [Q&A] [answered]
Hello, I've tried to play a sound when a new message appear but when I try to add a new highlight I don't know what to use in the "Pattern" field. Is there a list of patterns I can use? Basically I'm starting with streaming and my followers base isn't big at the time, so I want to play a sound for every message (ok, maybe with a 60 second cooldown so that I don't get spammed with sounds), but I don't know how to do it or if it's even possible. Can you help me with that? Thanks.

**Comment by KararTY** [comment, marked as answer]
That sounds like a very bad time, but if you really want to do that,
put in `.*` into Pattern and click on `Enable regex` checkbox.

To add a cooldown, make an audio clip that has silence at the end. Up to you how long that silence is.

Good luck!

**Reply by apnof** [reply]
Great. Came here with the exact question. 
As a newbie with average of ~0,5 viewers per stream, I want to notice when someone is writing.
Of course, the moment there is normal chat activity you'll want to disable this funktion.
Thanks

**Reply by apnof** [reply]
hm, unfortunately, it doesn't work.
What did I mess up?
![image](https://github.com/Chatterino/chatterino2/assets/945269/fb65354b-2c43-41e7-9906-7146b99f7523)

**Reply by Mm2PL** [reply]
Everything should be fine. You can't ping yourself if that's how you're testing it.

**Reply by apnof** [reply]
No, I asked friends to write in chat while I was live on twitch.
Weird. One thing might be the audio file, i used an old wav, might have been a unlucky format. Will try again with a mp3.

Just to be sure that I did understand right: 
With this setting from the screenshot, chatterino should play a sound whenever there is any message in any tab, right?

Thanks for helping btw.

edit:
I might have found something (test pending):
![image](https://github.com/Chatterino/chatterino2/assets/945269/ea6dc179-639c-4553-ada1-11b9bdc94c34)

**Reply by Mm2PL** [reply]
> Just to be sure that I did understand right:
> With this setting from the screenshot, chatterino should play a sound whenever there is any message in any tab, right?

Correct. 

> I might have found something 

I didn't take streamer mode into consideration. Disabling this should make it work?

**Reply by apnof** [reply]
wooh, confirm, streamer mode was blocking it.
**It works now.**

Sound is completely overdriven, but nothing I can fix with changing the volume within the sound file itself.
(And it doesn't matter, if it sounds awful as long as i get the notification)

Thanks again
